.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Feature request
+about: Got a cool idea you would like implemented? Share it to us!
+title: ''
+labels: enhancement
+assignees: ''
+
+---
+
+### Summary
+<!-- Small summary of the feature -->
+
+### Problem
+<!-- What problem will this feature solve (if any)? -->
+
+### Ideal implementation
+<!-- How should this feature be implemented? -->
```

Comment:
Just wondering if how a feature should be implemented is more of a code design question here, interface design or a mix of the two plus/or maybe something else. If this was intentionally left open-ended to allow for both then i understand that just want to know the intentions of this line

Reply:
This was more an open question, my idea was for them to link to some resources we could use or provide some sample code or the ideal way they would like to use it. This is left open-ended on purpose to allow anybody to express how they feel this should get implemented in any way, code wise or design wise